### PR TITLE
post-merge updates: staking content

### PR DESCRIPTION
**Pull request by jmcook1186**
## Description

updates staking content ready for post-merge

## Related Issue

#7075

### CODE REVIEW

File: src/content/staking/pools/index.md [hunk 1]
```diff
@@ -67,17 +67,9 @@ Typically ERC-20 liquidity tokens are issued to stakers that represents the valu
...
-Alternatively, pools that utilize an ERC-20 liquidity token allow users to trade this token in the open market, effectively allowing you to "withdraw" without actually removing ETH from the Beacon Chain.
-</ExpandableCard>
-
-<ExpandableCard title="Do pooled stakers need to do anything for The Merge?">
-Pooled stakers <strong>do not need to do anything to prepare for The Merge</strong>.
-
-However, as The Merge approaches, be on high alert for scammers. **You do not need to upgrade your ETH or staked ETH tokens** for the transition to proof-of-stake.
-
-Learn more about [The Merge](/upgrades/merge/)
+Alternatively, pools that utilize an ERC-20 liquidity token allow users to trade this token in the open market, effectively allowing you to "withdraw" without actually removing ETH from the staking contract.
```
Comment (wackerow) [inline src/content/staking/pools/index.md:72]: Just a thought
```suggestion
Alternatively, pools that utilize an ERC-20 liquidity token allow users to trade this token in the open market, allowing you to sell your staking position, effectively "withdrawing" without actually removing ETH from the staking contract.
```

File: src/content/staking/saas/index.md [hunk 2]
```diff
@@ -76,21 +76,9 @@ All of these keys can always be regenerated in a reproducible manner using your
...
-The Merge will <em>not</em> enable the ability to withdraw your stake or protocol rewards; this feature is planned for the Shanghai upgrade, which will follow the Merge by an estimated six months to a year.
-</ExpandableCard>
-
-<ExpandableCard title="Do SaaS stakers need to do anything for The Merge?">
-SaaS stakers <strong>do not need to do anything to prepare for The Merge</strong>.
-
-There are a few things node operators must attend to for this upgrade. Check with your staking provider for assurance their systems are ready to go.
-
-Learn more about [The Merge](/upgrades/merge/)
 </ExpandableCard>
```
Comment (wackerow) [inline src/content/staking/saas/index.md:83]: image

Ended up with an extra closing tag here

```suggestion
```

File: src/content/staking/solo/index.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ As much as we wish that solo staking was accessible and risk free to everyone, t
     <p style={{marginTop: "1rem"}}><a href="/security">Ethereum security and scam prevention</a></p>
   </ExpandableCard>
   <ExpandableCard title="No withdrawing (for now)" eventCategory="SoloStaking" eventName="clicked no withdrawing">
-    Withdrawing staked ETH or rewards from a validator balance is not yet supported. Support for withdrawals are planned for the Shanghai upgrade following The Merge. You should anticipate your ETH being locked for at least one-to-two years. After the Shanghai upgrade you will be able to freely withdraw portions or all of your stake if you wish.
+    Withdrawing staked ETH or rewards from a validator balance is not yet supported. Support for withdrawals are planned for the Shanghai upgrade. You should anticipate your ETH being locked for at least one-to-two years. After the Shanghai upgrade you will be able to freely withdraw portions or all of your stake if you wish.
```
Comment (wackerow) [inline src/content/staking/solo/index.md:61]: Just a thought to infer that this upgrade is coming next
```suggestion
    Withdrawing staked ETH or rewards from a validator balance is not yet supported. Support for withdrawals are planned for the upcoming Shanghai upgrade. You should anticipate your ETH being locked for at least one-to-two years. After the Shanghai upgrade you will be able to freely withdraw portions or all of your stake if you wish.
```

**Review by wackerow — CHANGES_REQUESTED** [review]
@jmcook1186 Couple suggestions, one of which required (need to remove that lonely closing tag). Otherwise looks good!

File: src/content/staking/pools/index.md [hunk 4]
```diff
@@ -67,17 +67,9 @@ Typically ERC-20 liquidity tokens are issued to stakers that represents the valu
 </ExpandableCard>
 
 <ExpandableCard title="When can I withdraw my stake?">
-Currently, withdrawing funds from a validator on the Beacon Chain is not possible, which currently limits the ability to actually <em>redeem</em> your liquidity token for the ETH rewards locked in the consensus layer.
+Currently, withdrawing funds from an Ethereum validator is not possible, which limits the ability to actually <em>redeem</em> your liquidity token for the ETH rewards locked in the consensus layer.
```
Comment (minimalsm) [inline src/content/staking/pools/index.md:70]:
```suggestion

Currently, withdrawing funds from an Ethereum validator is not possible, which limits the ability to actually _redeem_ your liquidity token for the ETH rewards locked in the consensus layer.

```

File: src/content/staking/solo/index.md [hunk 5]
```diff
@@ -130,7 +126,7 @@ Have a suggestion for a staking tool we missed? Check out our [product listing p
 These are a few of the most common questions about staking that are worth knowing about.
 
 <ExpandableCard title="What is a validator?">
-A <em>validator</em> is a virtual entity that lives on the Beacon Chain and participates in the consensus of the Ethereum protocol. Validators are represented by a balance, public key, and other properties. A <em>validator client</em> is the software that acts on behalf of the validator by holding and using its private key. A single validator client can hold many key pairs, controlling many validators.
+A <em>validator</em> is a virtual entity that lives on Ethereum and participates in the consensus of the Ethereum protocol. Validators are represented by a balance, public key, and other properties. A <em>validator client</em> is the software that acts on behalf of the validator by holding and using its private key. A single validator client can hold many key pairs, controlling many validators.
```
Comment (minimalsm) [inline src/content/staking/solo/index.md:129]:
```suggestion

A _validator_ is a virtual entity that lives on Ethereum and participates in the consensus of the Ethereum protocol. Validators are represented by a balance, public key, and other properties. A <em>validator client</em> is the software that acts on behalf of the validator by holding and using its private key. A single validator client can hold many key pairs, controlling many validators.

```

**Review by minimalsm — APPROVED** [review]

**Pull request merged**